integration/test_project_management.py

```diff
@@ -34,22 +44,144 @@ def test_android_app(default_app):
         package_name=TEST_APP_PACKAGE_NAME, display_name=TEST_APP_DISPLAY_NAME_PREFIX)
 
 
-def test_get_android_app_metadata(test_android_app, project_id):
+@pytest.fixture(scope='module')
+def test_ios_app(default_app):
```

Comment:
Lets not prefix fixtures with `test_`. That's confusing. Lets just call this `ios_app`? Same goes for `test_android_app` above.

Reply:
Done.